### PR TITLE
chore: stop using node urls with kubectl-mayastor MQ-2537

### PR DESCRIPTION
**Pull request by blaisedias**
With VPN access kubectl-mayastor just works if KUBECONFIG is
set up correctly.

This simplifies the control plane code interfacing with the restful
control-plane.

Rip out most of the offending bits.
But retain the use of IP addresses, to communicate with the
restful control plane for replica information.
Uses master node IP addresses now instead of worker nodes,
this will change when MQ-2319 is merged.

### CODE REVIEW

**Comment by blaisedias** [comment]
bors try

**Review by rohan2794 — APPROVED** [review]
LGTM

File: src/common/k8stest/msv.go [hunk 1]
```diff
@@ -15,7 +15,15 @@ var once sync.Once
 // Facilitates abstraction without introducing a dependency on k8stest
 func EnsureNodeAddressesAreSet() {
 	once.Do(func() {
-		controlplane.SetIpNodeAddresses(GetMayastorNodeIPAddresses())
+		var masterNodeIPAddrs []string
+		// FIXME: for now ignore the error and emit a warning an imminent change rids us of this function
```
Comment (chriswldenyer) [inline src/common/k8stest/msv.go:19]: The comment says that this emits a warning on error, but it looks like it doesn't.
What is the error to ignore?

**Review by chriswldenyer — APPROVED** [review]
LGTM. One comment.

File: src/common/k8stest/msv.go (hunk 1 above)
Comment (blaisedias) [reply src/common/k8stest/msv.go:21]: Failure to list cluster nodes. Fixed.

Comment (blaisedias) [reply src/common/k8stest/msv.go:21]: https://github.com/mayadata-io/mayastor-e2e/pull/354/commits/c32f7de60baf26aa53bbf152b5e76de651a14c8e

**Comment by blaisedias** [comment]
bors merge

**Comment by blaisedias** [comment]
bors merge

**Review by payes — APPROVED** [review]

**Comment by blaisedias** [comment]
bors merge

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**